Create README.md (#1)
mdn/.github: profile/README.md

# 👋 Welcome, Bienvenida, 欢迎, Bienvenu, いらっしゃいませ, Receber

![github-profile](https://user-images.githubusercontent.com/10350960/166113119-629295f6-c282-42c9-9379-af2de5ad4338.png)

> MDN Web Docs is an open-source, collaborative project documenting Web platform technologies, including CSS, HTML, JavaScript, and Web APIs. We also provide extensive 👨🏿‍🎓 learning resources for beginning developers and students.

Reviewer: ```suggestion
MDN Web Docs is an open-source, collaborative project that documents web platform technologies, including CSS, HTML, JavaScript, and Web APIs. We also provide extensive 👨🏿‍🎓 learning resources for beginning developers and students.
```

## 🙌 MDN's mission is to provide a blueprint for a better internet and empower a new generation of developers and content creators to build it.

> The power of MDN Web Docs lies in its vast, vital community of active readers and contributors. Since 2005, approximately 45,000 contributors have created the documentation we know and love. 💗 Together, contributors have created over 45,000 documents that make up an up-to-date, comprehensive, and free resource for web developers worldwide. 🤯 In addition to English-language articles, over 35 volunteers lead translation and localization efforts for 🇨🇳 Chinese, 🇫🇷 French, 🇯🇵 Japanese, 🇰🇷 Korean, 🇵🇹 Portuguese, 🇷🇺 Russian, and 🇪🇸 Spanish.

Reviewer: ```suggestion
The power of MDN Web Docs lies in its vast community of active readers and contributors. Since 2005, approximately 45,000 contributors have created the documentation we know and love. Together, contributors have created over 45,000 documents that make up an up-to-date, comprehensive, and free resource for web developers worldwide. In addition to English-language articles, over 35 volunteers lead translation and localization efforts for 🇨🇳 Chinese, 🇫🇷 French, 🇯🇵 Japanese, 🇰🇷 Korean, 🇵🇹 Portuguese, 🇷🇺 Russian, and 🇪🇸 Spanish.
```

- By "Together", do we mean "In all"?
- Just a personal opinion to delete those two emojis - the flag ones are good to keep though.

### 🧑🏼‍🤝‍🧑  Be part of MDN Web Docs

> You can be part of MDN Web Docs, whether it be through ✍🏽 content contributions, ⚙️ engineering, or ↔️ translation work. The MDN Web Docs project welcomes contributions from everyone who shares our goals and wants to contribute healthily and constructively within our community. 🧘‍♂️

Reviewer: ```suggestion
You can be part of MDN Web Docs, whether it be through ✍🏽 content contributions, ⚙️ engineering, or ↔️ translation work. The MDN Web Docs project welcomes contributions from everyone who shares our goals and wants to contribute constructively and respectfully within our community. 🧘‍♂️
```

### ✉️ Get in touch

> You can communicate with the MDN Web Docs team and community through our Matrix channel, forums on Discourse, and discussions on GitHub.

Reviewer: ```suggestion
You can communicate with the MDN Web Docs team and community through our [Matrix channel](https://wiki.mozilla.org/Matrix), forums on [Discourse](https://discourse.mozilla.org/c/mdn/236), and discussions on [GitHub](https://github.com/mdn/mdn-community/discussions).
```
